lib/foreman_build_history/facet.rb

```diff
@@ -1,7 +1,2 @@
 Facets.register(BuildHistoryFacet) do
```

Comment:
Not exactly through facets, but through initializer:
https://github.com/theforeman/foreman/blob/develop/config/initializers/puppet.rb#L3